setup.py

```diff
@@ -49,6 +49,7 @@
         'jinja2==2.7.2',
         'retrying==1.3.3',
         'colorama==0.3.3',
+        'requests==2.7.0',
```

Comment:
It's the version that was already installed in my virtual environment because is required in other repositories like:  `cloudify-dsl-parser`, `cloudify-manager`, `cloudify-rest-client`, ...

In case we decide to go for the latest version, the requirement should probably be updated in all the related repositories to avoid requiring different versions in the same virtual environment.